server/handlers/auth-handler.js

```diff
@@ -0,0 +1,32 @@
+const { HeaderValidationError } = require('../errors')
+const httpStatus = require('http-status')
```

Comment:
Since `httpStatus` is defined in the global scope I think it is not good to define it as a generic or a compact name. For instance, `status` is a very generic name which can be useful to be used in inner scopes.